website/docs/content/writing-style/index.md

```diff
@@ -0,0 +1,306 @@
+---
+title: Writing Style
+caption: How HashiCorp structures writing, in a clear concise manner.
+description: How HashiCorp structures writing, in a clear concise manner.
+previewImage: assets/illustrations/content/writing-guidelines.jpg
+---
+
+Generally, HashiCorp follows the [2020 AP Stylebook](https://www.apstylebook.com/ap_stylebook) rules and guidelines, unless HashiCorp specific style is indicated.
+
+## Capitalization
+
+### Headings and labels
+
+Use sentence-case for titles, headings, labels, links, and buttons. Use title case for proper nouns and when referring to products (e.g., Nomad).
+
+!!! Do
+
+![Showing a title with sentence-case](/assets/content/writing-style/writing-style-capitlization-header-do.png)
+
+![Showing an input with a label using sentence-case and title case for a proper noun](/assets/content/writing-style/writing-style-capitlization-input-do.png)
+
+![Showing a link using sentence-case](/assets/content/writing-style/writing-style-capitlization-link-do.png)
+
+!!!
+
+!!! Dont
+
+![Showing a title with capital-case](/assets/content/writing-style/writing-style-capitlization-header-dont.png)
+
+![Showing an input with a label using title case](/assets/content/writing-style/writing-style-capitlization-input-dont.png)
+
+![Showing a link using sentence-case](/assets/content/writing-style/writing-style-capitlization-link-dont.png)
+
+!!!
+
+### Feature names
+
+Avoid capitalizing feature names or internal constructs as proper nouns.
+
+!!! Do
+
+Terraform Cloud includes a private module registry.
+
+A plan represents the execution plan of a run.
+
+Enable a Secrets Engine in Vault
+
+We’ll use the Vault namespaces feature.
+
+!!!
+
+!!! Dont
+
+Terraform Cloud includes a Private Module Registry.
+
+A Plan represents the execution plan of a Run.
+
+Enable a secrets engine in Vault
+
+We’ll use the Vault Namespaces feature.
+
+!!!
+
+**Capitalization rules for UI elements**
+
+Typically you will want to capitalize site or UI elements or sections when referring to them in content if they are capitalized in the UI:
+
+“You can set up module sharing from the Site Administration area of a Terraform Enterprise instance. The Registry subsection allows an organization to be chosen to which your current organization’s private module registry will be shared.”
```

Comment:
Should this be in a "Do" style

Reply:
I thought about this, but the way the page has been structured is do and don't are comparative. This would break pattern, but maybe overthinking it here.